autogen/README.md

```diff
@@ -25,6 +25,27 @@ If you are using these features with a private cluster, you will need to either:
 3. Include the external IP of your Terraform deployer in the `master_authorized_networks` configuration. Note that only IP addresses reserved in Google Cloud (such as in other VPCs) can be whitelisted.
 4. Deploy a [bastion host](https://github.com/terraform-google-modules/terraform-google-bastion-host) or [proxy](https://cloud.google.com/solutions/creating-kubernetes-engine-private-clusters-with-net-proxies) in the same VPC as your GKE cluster.
 
+## Private Cluster Update
```

Comment:
Update variants are not specific to private clusters. Rather, this should be conditional based on whether update variant is true.